README.md

```diff
@@ -40,6 +40,17 @@ In 2016 we're using a static site generator, [metalsmith](http://metalsmith.io),
 * `npm test` will run tests in /test/
   * TODO: unify tests, add more of them, integrate with circleCI
 
+### Deploying
+* [Build status is in Jenkins](https://ci.us-west.moz.works/view/viewsourceconf/)
+* Code merged to [https://github.com/mdn/viewsourceconf/tree/master](https://github.com/mdn/viewsourceconf/tree/master) will land on [staging](viewsourceconf-stage.us-west.moz.works) automatically, unless it breaks the build.
+* Don't do anything else until your code is merged to master and you have tested it in staging. Srsly.
+* Tags pushed to Mozilla's remote master will trigger a production build. To tag...
+  * (Have permissions to push code to the Mozilla remote master)
+  * Check out the [Mozilla remote master] (https://github.com/mdn/viewsourceconf/tree/master)
+  * Set an environment variable containing the name of the remote master: `export MOZ_GIT_REMOTE=upstream` (where 'upstream' is the name of the remote master).
+  * Run the tagging script conveniently located at the root of the repo. It is probably not set to be executable, so `. tag-release.sh`
```

Comment:
I ran `. tag-release.sh --push` I think that has to be in there or we need to explicitly say to push.